sharktank/sharktank/ops/qconv_impls.py

```diff
@@ -112,8 +112,11 @@ def qconv2d_tensor_scaled_integer(
     stride = _expand_int_to_2_tuple(stride)
     padding = _expand_int_to_2_tuple(padding)
     dilation = _expand_int_to_2_tuple(dilation)
+    extended_list = [item for item in padding for _ in range(2)]
```

Comment:
Please name `extended_padding_list` (or something that indicates this is padding related)